src/App.js

```diff
@@ -46,31 +47,55 @@ function App() {
           <option value="success">Success</option>
           <option value="error">Error</option>
         </select>
+        <select
+          className="block appearance-none mb-6 w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+          id="lang"
+          onChange={({ target }) => {
+            setPosition(target.value);
+            setActive(false);
+          }}
+          value={position}
+        >
+          <option value="default">Select Notification Position</option>
+          <option value="left">Left</option>
+          <option value="right">Right</option>
+          <option value="bleft">Bottom Left</option>
+          <option value="bright">Bottom Right</option>
+        </select>
         <textarea
-          class="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+          className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
           cols="30"
           rows="10"
           value={message}
           onChange={(e) => setMessage(e.currentTarget.value)}
           placeholder="Add a message to display in the component"
         ></textarea>
 
-        <div class="mt-6">
-          <span class="block w-full rounded-md shadow-sm">
+        <div className="mt-6">
+          <span className="block w-full rounded-md shadow-sm">
             <button
               type="submit"
               onClick={() => {
                 console.log("here");
                 setActive(true);
               }}
-              class="w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-500 focus:outline-none focus:border-indigo-700 focus:shadow-outline-indigo active:bg-indigo-700 transition duration-150 ease-in-out"
+              className="w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-500 focus:outline-none focus:border-indigo-700 focus:shadow-outline-indigo active:bg-indigo-700 transition duration-150 ease-in-out"
             >
               Show Notification
             </button>
           </span>
         </div>
 
-        {active ? <Toast type={type} message={message} duration={3000} /> : ""}
+        {active ? (
+          <Toast
+            type={type}
+            message={message}
+            position={position}
+            duration={3000}
+          />
```

Comment:
can we make this into a boolean check instead of returning the `""` when it's not active? 
so something like this `active && <Toast />`. It's cleaner this way and we get to remove any unwanted elements from the DOM.